src/common/config/store.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"os"
+)
+
+// Store ...
+type Store interface {
+	Get(key string) (*ConfigureValue, error)
+	Set(key string, value ConfigureValue) error
+}
+
+// Driver ...
```

Comment:
it's better to place the Driver interface into the localstore?

Reply:
Will there be problem if we put Driver interface in the `driver` package and put the Store interface in the `store` package?